parts/linux/cloud-init/artifacts/cse_start.sh

```diff
@@ -0,0 +1,11 @@
+/bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1
+EXIT_CODE=$?
+systemctl --no-pager -l status kubelet >> /var/log/azure/cluster-provision-cse-output.log 2>&1
+OUTPUT=$(cat /var/log/azure/cluster-provision-cse-output.log | head -n 30)
+JSON_STRING=$( jq -n \
+                  --arg ec "$EXIT_CODE" \
+                  --arg op "$OUTPUT" \
+                  --arg er "" \
```

Comment:
I don't get this part, we always make er empty?

Reply:
currently we don't have a very good way to get error message from error code, but in future we will add detailed error message here.

Reply:
this's a placeholder for now